From 9403554020974f5a26ad6682b7bd27a4b6fe99e2 Mon Sep 17 00:00:00 2001
From: Oscar Lesta <oscar.lesta@gmail.com>
Date: Tue, 4 Feb 2025 04:39:40 -0300
Subject: Fix calling Python on Haiku.

Haiku's "default Python" package do not currently provide /bin/python.

diff --git a/tests/run-doctests.py b/tests/run-doctests.py
index 7cc39e2..4921614 100644
--- a/tests/run-doctests.py
+++ b/tests/run-doctests.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import doctest
 import os
diff --git a/tests/setup.sh b/tests/setup.sh
index 4ecf32f..33a7539 100644
--- a/tests/setup.sh
+++ b/tests/setup.sh
@@ -6,7 +6,7 @@
 
 # The $PYTHON environment variable should be set when running this test
 # from Python.
-[ -n "$PYTHON" ] || PYTHON="`which python`"
+[ -n "$PYTHON" ] || PYTHON="`which python3`"
 [ -n "$PYTHONPATH" ] || PYTHONPATH="$TESTDIR/.." && export PYTHONPATH
 if [ -n "$COVERAGE" ]; then
   if [ -z "$COVERAGE_FILE" ]; then
-- 
2.45.2

